<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetCanvasDialog</class>
 <widget class="QDialog" name="ResetCanvasDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reset Canvas</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="heightLabel">
       <property name="text">
        <string>Height:</string>
       </property>
       <property name="buddy">
        <cstring>heightLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="widthLabel">
       <property name="text">
        <string>Width:</string>
       </property>
       <property name="buddy">
        <cstring>widthLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="heightLineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="widthLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>widthLineEdit</tabstop>
  <tabstop>heightLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResetCanvasDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResetCanvasDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
